pkg/providers/tinkerbell/validator.go

```diff
@@ -245,3 +254,17 @@ func sumWorkerNodeCounts(nodes []v1alpha1.WorkerNodeGroupConfiguration) int {
 	}
 	return requestedNodesCount
 }
+
+// validateAddressWithPort validates address is a hostname:port combination. The port is required.
+func validateAddressWithPort(address string) error {
+	host, _, err := net.SplitHostPort(address)
```

Comment:
we should validate the port as well maybe using this https://github.com/aws/eks-anywhere/blob/main/pkg/networkutils/networkutils.go#L9-L12

Reply:
Done.